assets/src/blocks/GuestBook/GuestBookFrontend.js

```diff
@@ -1,15 +1,16 @@
+/* eslint-disable jsx-a11y/iframe-has-title */
 export const GuestBookFrontend = () => {
   const buildURL = () => {
     const hostname = window.location.hostname;
-    const pathname = window.location.pathname.slice(1);
-    const params = new URLSearchParams(window.location.search);
-    const storyID = params.get('id');
-    return `https://maps.greenpeace.org/maps/gpint/50th_guestbook/?origin_host=${hostname}&origin_path=${encodeURIComponent(pathname)}${storyID ? `&id=${storyID}` : ''}`;
-  }
+    const pathname = window.location.pathname.slice( 1 );
+    const params = new URLSearchParams( window.location.search );
+    const storyID = params.get( 'id' );
+    return `https://maps.greenpeace.org/maps/gpint/50th_guestbook/?origin_host=${ hostname }&origin_path=${ encodeURIComponent( pathname ) }${ storyID ? `&id=${ storyID }` : '' }`;
+  };
 
   return (
     <p>
-      <iframe src={buildURL()} width='100%' height={700} frameBorder='0' />
+      <iframe src={ buildURL() } width="100%" height={ 700 } frameBorder="0" />
```

Comment:
I'm pretty sure that we could add a title here instead of using `/* eslint-disable jsx-a11y/iframe-has-title */`, even something super simple like `title="Guestbook"`

Reply:
Done.